menu.rb

```diff
@@ -0,0 +1,25 @@
+# Elsa's random menu generator
+# Due: 150511
+# Breakfast + Lunch + 1 Dinner feature: Expand your solution to ensure that no descriptive term in a menu item is ever repeated.
+
+
+# I'm not a foodie, so ha
+food1 = ["spooky", "cumbersome", "alarming", "damp", "furry", "milky", "unnatural", "hissing", "lewd", "jumping", "wiggly"]
```

Comment:
I appreciate your use of white space after the commas here - nice and easy to read